unofficial/c511027787.lua

```diff
@@ -0,0 +1,102 @@
+--ＲＲ－レディネス (アニメ)
+--Raidraptor - Readiness (Anime)
+--Made by When
+local s,id=GetID()
+function s.initial_effect(c)
+    --Activate (if you would take battle damage)
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCode(EVENT_PRE_DAMAGE_CALCULATE)
+	e1:SetCondition(s.condition1)
+	e1:SetOperation(s.activate1)
+	c:RegisterEffect(e1)
+	--Activate (if you would take effect damage)
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(id,0))
+	e2:SetProperty(EFFECT_FLAG_DAMAGE_STEP)
+	e2:SetType(EFFECT_TYPE_ACTIVATE)
+	e2:SetCode(EVENT_CHAINING)
+	e2:SetCondition(s.condition2)
+	e2:SetOperation(s.activate2)
+	c:RegisterEffect(e2)
+	--Banish From GY
+	local e3=Effect.CreateEffect(c)
+	e3:SetDescription(aux.Stringid(id,1))
+	e3:SetProperty(EFFECT_FLAG_DAMAGE_STEP)

```

Comment:
This effect cannot be used during the damage step, so no EFFECT_FLAG_DAMAGE_STEP